docs/articles/documentation/extending-testcafe/custom-reporter/index.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: docs
+title: Custom Reporter Plugin
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/
+---
+# Custom Reporter Plugin
+
+TestCafe has a number of [built-in reporters](/testcafe/documentation/using-testcafe/common-concepts/reporters/) to generate test reports in different formats.
+You can also create a **custom reporter** that will output information in your own format and style.
+For this purpose, you can use the [TestCafe reporter generator](https://github.com/DevExpress/generator-testcafe-reporter).
+The generator will scaffold out a reporter plugin, so that you only need to write a few lines of code.
+
+For more information, refer the following topics.
+
+* [Creating Reporter Plugin](/testcafe/documentation/extending-testcafe/custom-reporter-plugin/creating-reporter-plugin/)
+* [Helper Methods](/testcafe/documentation/extending-testcafe/custom-reporter-plugin/helper-methods)
```

Comment:
I am immediately getting afraid that we are losing integrity.
Probably, we'd better add context than remove this link.
Something like

```
For more information on how to implement a plugin, refer to

* [Creating Reporter Plugin](/testcafe/documentation/extending-testcafe/custom-reporter-plugin/creating-reporter-plugin/)

When creating the reporter plugin, you will need to use some helpers to format the report. You can find the reference in the following topic:

* [Helper Methods](/testcafe/documentation/extending-testcafe/custom-reporter-plugin/helper-methods)
```